### ISSUE TITLE
[Bug]: urlpattern not define 

### ISSUE THREAD

**Issue by deku-m** [Onemind-Services-LLC/netbox-metatype-importer#10]
### NetBox plugin version

v0.0.3

### NetBox version

v3.4.4

### Steps to Reproduce

After updating the docker environment with latest release i am not able to start the netbox env.
In the logging the docker env is showing that urlpatterns attribute is missing. See attached log under "Observed Behavior"

When disabling the plugin everything is running again.

### Expected Behavior

Starting netbox and running the environment.

### Observed Behavior

```
2023-02-03 07:37:50 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:37:50 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:37:50 ⏳ Waiting on DB... (0s / 30s)
2023-02-03 07:37:55 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:37:55 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:37:55 ⏳ Waiting on DB... (3s / 30s)
2023-02-03 07:37:59 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:37:59 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:37:59 ⏳ Waiting on DB... (6s / 30s)
2023-02-03 07:38:04 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:38:04 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:38:04 ⏳ Waiting on DB... (9s / 30s)
2023-02-03 07:38:09 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:38:09 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:38:09 ⏳ Waiting on DB... (12s / 30s)
2023-02-03 07:38:14 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:38:14 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:38:14 ⏳ Waiting on DB... (15s / 30s)
2023-02-03 07:38:18 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:38:18 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:38:18 ⏳ Waiting on DB... (18s / 30s)
2023-02-03 07:38:23 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:38:23 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:38:23 ⏳ Waiting on DB... (21s / 30s)
2023-02-03 07:38:28 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:38:28 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:38:28 ⏳ Waiting on DB... (24s / 30s)
2023-02-03 07:38:33 ImportError: Module "netbox_metatype_importer.api.urls" does not define a "urlpatterns" attribute/class
2023-02-03 07:38:33 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-03 07:38:33 ⏳ Waiting on DB... (27s / 30s)
2023-02-03 07:38:36 ❌ Waited 30s or more for the DB to become ready.
```

**Comment by abhi1693** [comment]
We will take a look at this. We have tested this only until 3.4.3

**Comment by deku-m** [comment]
I see it is closed but the problem still exist for me. I see that there is a pull request attached for it maybe that will fix it?

**Comment by abhi1693** [comment]
The PR is enhancement and doesn't affect the bug. Make sure you are using v3.4.4 of netbox

**Comment by deku-m** [comment]
Im running 3.4.4 but still getting the following after building and starting the docker again:

```
2023-02-13 10:20:09 django.core.exceptions.ImproperlyConfigured: Application labels aren't unique, duplicates: netbox_metatype_importer
2023-02-13 10:20:09 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-13 10:20:09 ⏳ Waiting on DB... (21s / 30s)
2023-02-13 10:20:12 django.core.exceptions.ImproperlyConfigured: Application labels aren't unique, duplicates: netbox_metatype_importer
2023-02-13 10:20:12 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-13 10:20:12 ⏳ Waiting on DB... (24s / 30s)
2023-02-13 10:20:16 django.core.exceptions.ImproperlyConfigured: Application labels aren't unique, duplicates: netbox_metatype_importer
2023-02-13 10:20:16 [ Use DB_WAIT_DEBUG=1 in netbox.env to print full traceback for errors here ]
2023-02-13 10:20:16 ⏳ Waiting on DB... (27s / 30s)
2023-02-13 10:20:19 ❌ Waited 30s or more for the DB to become ready.
```

**Comment by abhi1693** [comment]
Please raise a separate issue. But it'd be helpful to get a full stack trace as this issue is not reproducible on my Netbox
